### PR TITLE
Parameterize cluster dir

### PR DESCRIPTION
**Pull request by qinqon**

### CODE REVIEW

File: cluster-up/hack/common.sh [hunk 1]
```diff
@@ -15,6 +15,8 @@ if [ -z "$KUBEVIRTCI_CONFIG_PATH" ]; then
     )"
 fi
 
+
+KUBEVIRTCI_CLUSTER_PATH=${KUBEVIRTCI_CLUSTER_PATH:-${KUBEVIRTCI_PATH}/cluster-up/cluter}
```
Comment (oshoval) [inline cluster-up/hack/common.sh]: cluster or cluter ?

Comment (cynepco3hahue) [reply cluster-up/hack/common.sh]: should be cluster

**Pull request merged**